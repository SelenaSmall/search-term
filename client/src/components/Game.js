import React from 'react'
import { Link } from 'react-router-dom'
import './game.css';

const Game = () => (
    <div>
      <section id="top-region">
        <menu className="status-menu">
          <ul>
            <li className="status-menu-item status">
              <span className="status-menu-item-label">Status</span>
              <span className="status-menu-item-value"></span>
            </li>
            <li className="status-menu-item score">
              <span className="status-menu-item-label">Score</span>
              <span className="status-menu-item-value"></span>
            </li>
            <li className="status-menu-item timer">
              <span className="status-menu-item-label">Time</span>
              <span className="status-menu-item-value"></span>
            </li>
          </ul>
        </menu>
      </section>

      <section id="middle-region">
        <div className="game-main-container text-center">
          <div>
            <h1>Welcome to the Game</h1>
            <Link to="/round/1" className="game-button start-game-button">Start Game</Link>
          </div>
        </div>
      </section>

      <section id="bottom-region">
        <menu className="twitter-menu">
          <ul>
            <li className="twitter-menu-item twitter-menu-item-left status">
              <a href="http://twitter.com/intent/user?screen_name=selenasmall88">
                <img className="twitter-menu-item-avatar" src="/avatar-selena.jpg" alt="selena-small"></img>
                <span>@selenasmall88</span>
              </a>
            </li>
            <li className="twitter-menu-item status">
              <a href="http://twitter.com/intent/user?screen_name=saramic">
                <img className="twitter-menu-item-avatar" src="/avatar-michael.jpg" alt="michael-milewski"></img>
                <span>@saramic</span>
              </a>
            </li>
          </ul>
        </menu>
      </section>
    </div>
)

export default Game