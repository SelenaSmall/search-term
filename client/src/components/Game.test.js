import React from 'react';
import ReactDOM from 'react-dom';
import Game from './Game';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Game />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('snapshot testing', () => {
  const output = shallow(<Game />)
  expect(shallowToJson(output)).toMatchSnapshot()
})