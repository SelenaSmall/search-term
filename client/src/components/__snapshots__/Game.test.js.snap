// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing 1`] = `
<div>
  <section
    id="top-region"
  >
    <menu
      className="status-menu"
    >
      <ul>
        <li
          className="status-menu-item status"
        >
          <span
            className="status-menu-item-label"
          >
            Status
          </span>
          <span
            className="status-menu-item-value"
          />
        </li>
        <li
          className="status-menu-item score"
        >
          <span
            className="status-menu-item-label"
          >
            Score
          </span>
          <span
            className="status-menu-item-value"
          />
        </li>
        <li
          className="status-menu-item timer"
        >
          <span
            className="status-menu-item-label"
          >
            Time
          </span>
          <span
            className="status-menu-item-value"
          />
        </li>
      </ul>
    </menu>
  </section>
  <section
    id="middle-region"
  >
    <div
      className="game-main-container text-center"
    >
      <div>
        <h1>
          Welcome to the Game
        </h1>
        <Link
          className="game-button start-game-button"
          replace={false}
          to="/round/1"
        >
          Start Game
        </Link>
      </div>
    </div>
  </section>
  <section
    id="bottom-region"
  >
    <menu
      className="twitter-menu"
    >
      <ul>
        <li
          className="twitter-menu-item twitter-menu-item-left status"
        >
          <a
            href="http://twitter.com/intent/user?screen_name=selenasmall88"
          >
            <img
              alt="selena-small"
              className="twitter-menu-item-avatar"
              src="https://twitter.com/selenasmall88/profile_image?size=normal"
            />
            <span>
              @selenasmall88
            </span>
          </a>
        </li>
        <li
          className="twitter-menu-item status"
        >
          <a
            href="http://twitter.com/intent/user?screen_name=saramic"
          >
            <img
              alt="michael-milewski"
              className="twitter-menu-item-avatar"
              src="https://twitter.com/saramic/profile_image?size=normal"
            />
            <span>
              @saramic
            </span>
          </a>
        </li>
      </ul>
    </menu>
  </section>
</div>
`;
